satpy/writers/core/image.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, name=None, filename=None, base_dir=None, enhance=None, **kwar
 
                     {platform_name}_{sensor}_{name}_{start_time:%Y%m%d_%H%M%S}.tif
 
-            base_dir (str):
```

Comment:
This was an attempt to make my IDE less dumb about creating writers where it kept complaining that `base_dir` should be a `dict` or something...it didn't work, but this change in the type annotation is accurate from what I can tell.